### PR TITLE
project strcture modified

### PR DESCRIPTION
**Pull request by Kajal0823**

### CODE REVIEW

File: src/components/Complain.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    const displaystatus = (index) => status1.map((d, i) => {
+      if (i == index) {
+        return d.count
+      }
+    })
+
+
+    const displayData = data.map((data1, i) => {
+      for (i = 0; i <= data1.length; i++) {
```
Comment (dhananjayc) [inline src/components/Complain.js:76]: Remove this `for loop` as its not doing anything here

File: src/components/login.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { withRouter } from 'react-router'
+import { fetchLogin, loginSucess, logoutUser } from "../actions/loginAction";
+import browserHistory from 'react-router'
+import Profile from '../pages/Profile';
+import { bindActionCreators, compose } from 'redux'
+
+
+class login extends Component {
+
+  componentDidMount() {
```
Comment (dhananjayc) [inline src/components/login.js:12]: Remove this method as we are not doing anything inside this function

File: src/components/login.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    super(props)
+    this.state = {
+      username: '',
+      password: '',
+    }
+  }
+  componentDidUpdate(pp, ps, ss) {
+
+  }
+  // loginSucess() {
```
Comment (dhananjayc) [inline src/components/login.js:29]: Remove commented code

File: src/reducers/apiReducer.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+const initalState = {
+  url = "",
+  method = "POST",
+  data = null,
+  accessToken = null,
+
+}
+const apiReducer = (state = initalState, action) => {
+  console.log('loginaction', action)
```
Comment (dhananjayc) [inline src/reducers/apiReducer.js:16]: Remove `console.log`

File: src/actions/loginAction.js [hunk 5]
```diff
@@ -1,78 +1,59 @@
+import { prepareRequestBody, prepareUrl, prepareAPICall } from '../Utilites/utils'
+import { LOGIN_URL } from '../constants/login'
 export const FETCH_LOGIN_BEGIN = 'FETCH_LOGIN_BEGIN';
```
Comment (dhananjayc) [inline src/actions/loginAction.js:3]: Move this constatnts in `constants` folder

File: src/actions/listAction.js [hunk 6]
```diff
@@ -1,113 +1,71 @@
-const token1 = localStorage.getItem("jwtToken");
+import { COMPLAIN_LIST_URL, COMPLAIN_COUNT_URL, COMPLAIN_STATUS_URL } from '../constants/complainList'
 
+import { prepareRequestBody, prepareRequestWithOutBody, prepareUrl, prepareAPICall } from '../Utilites/utils'
 export const FETCH_LIST_BEGIN = 'FETCH_LIST_BEGIN';
```
Comment (dhananjayc) [inline src/actions/listAction.js:4]: Move this constants in `constants` folder

**Review by dhananjayc — COMMENTED** [review]
Good Job! Please do this changes in next PR :)

**Comment by Kajal0823** [comment]
ok sir13.12.2019, 13:10, "Dhananjay C" <notifications@github.com>:@dhananjayc commented on this pull request.

Good Job! Please do this changes in next PR :)

In src/components/Complain.js:
> +    if (error) {
+      return <div>Error! {error.message}</div>;
+    }
+    if (loading) {
+      return <div>Loading...</div>;
+    }
+
+    const displaystatus = (index) => status1.map((d, i) => {
+      if (i == index) {
+        return d.count
+      }
+    })
+
+
+    const displayData = data.map((data1, i) => {
+      for (i = 0; i <= data1.length; i++) {

Remove this for loop as its not doing anything here



In src/components/login.js:
> @@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+import { fetchLogin, loginSucess, logoutUser } from "../actions/loginAction";
+import browserHistory from 'react-router'
+import Profile from '../pages/Profile';
+import { bindActionCreators, compose } from 'redux'
+
+
+class login extends Component {
+
+  componentDidMount() {

Remove this method as we are not doing anything inside this function



In src/components/login.js:
> +    // if (username !== null || password !== null) {
+    //this.props.dispatch(loginSucess())
+    // }
+
+  };
+  constructor(props) {
+    super(props)
+    this.state = {
+      username: '',
+      password: '',
+    }
+  }
+  componentDidUpdate(pp, ps, ss) {
+
+  }
+  // loginSucess() {

Remove commented code



In src/reducers/apiReducer.js:
> +import {
+  FETCH_API,
+  GET,
+  POST
+
+} from '../actions/apiActions';
+
+const initalState = {
+  url = "",
+  method = "POST",
+  data = null,
+  accessToken = null,
+
+}
+const apiReducer = (state = initalState, action) => {
+  console.log('loginaction', action)

Remove console.log



In src/actions/loginAction.js:
> @@ -1,78 +1,59 @@
+import { prepareRequestBody, prepareUrl, prepareAPICall } from '../Utilites/utils'
+import { LOGIN_URL } from '../constants/login'
 export const FETCH_LOGIN_BEGIN = 'FETCH_LOGIN_BEGIN';

Move this constatnts in constants folder



In src/actions/listAction.js:
>  
+import { prepareRequestBody, prepareRequestWithOutBody, prepareUrl, prepareAPICall } from '../Utilites/utils'
 export const FETCH_LIST_BEGIN = 'FETCH_LIST_BEGIN';

Move this constants in constants folder

**Pull request merged**